CHANGELOG.md

```diff
@@ -5,6 +5,33 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Removed
+* ESP-IDF v4.3 support, including mostly conditional compilations. (#420)
+### Changed
+* **ESP-IDF v4.4 is now considered deprecated.** Please start upgrading to ESP-IDF v5.
+* rmt: `Symbol` now derives `Clone` and `Copy` (#386)
+* GPIO20 is now included on esp32. Notice that not every esp32 has a physical GPIO20. (#370)
+* i2s: Compatibility with ESP-IDF v5.3 (pre-release) - small internal adjustments to i2s (#419)
+* ledc: now it can also be used with high_speed channels on esp32. (#424)
+* dependacy: updated embassy-sync to 0.6. (#437)
```

Comment:
"Nit": "dependency". And BTW a `[Breaking change]`.

Reply:
done